libsolidity/analysis/TypeChecker.cpp

```diff
@@ -2416,79 +2289,66 @@ void TypeChecker::typeCheckFunctionGeneralChecks(
 	std::vector<ASTPointer<ASTString>> const& argumentNames = _functionCall.names();
 
 	// Check number of passed in arguments
-	if (
-		arguments.size() < parameterTypes.size() ||
-		(!isVariadic && arguments.size() > parameterTypes.size())
-	)
+	if (arguments.size() < parameterTypes.size() || (!isVariadic && arguments.size() > parameterTypes.size()))
 	{
-		bool const isStructConstructorCall =
-			functionCallKind == FunctionCallKind::StructConstructorCall;
-
-		auto [errorId, description] = [&]() -> std::tuple<ErrorId, std::string> {
-			std::string msg = isVariadic ?
-				"Need at least " +
-				toString(parameterTypes.size()) +
-				" arguments for " +
-				std::string(isStructConstructorCall ? "struct constructor" : "function call") +
-				", but provided only " +
-				toString(arguments.size()) +
-				"."
-				:
-				"Wrong argument count for " +
-				std::string(isStructConstructorCall ? "struct constructor" : "function call") +
-				": " +
-				toString(arguments.size()) +
-				" arguments given but " +
-				std::string(isVariadic ? "need at least " : "expected ") +
-				toString(parameterTypes.size()) +
-				".";
+		bool const isStructConstructorCall = functionCallKind == FunctionCallKind::StructConstructorCall;
+
+		auto [errorId, description] = [&]() -> std::tuple<ErrorId, std::string>
+		{
+			std::string msg = isVariadic
+								  ? "Need at least " + toString(parameterTypes.size()) + " arguments for "
+										+ std::string(isStructConstructorCall ? "struct constructor" : "function call")
+										+ ", but provided only " + toString(arguments.size()) + "."
+								  : "Wrong argument count for "
+										+ std::string(isStructConstructorCall ? "struct constructor" : "function call")
+										+ ": " + toString(arguments.size()) + " arguments given but "
+										+ std::string(isVariadic ? "need at least " : "expected ")
+										+ toString(parameterTypes.size()) + ".";
```

Comment:
This is another case, where you need to fiddle with spaces to align things the new way. I'm not even sure what it is trying to align things with (why aren't `?` and `:` aligned with `=`)?

It shifts code to the right for no good reason which will make it taller and narrower - I guess in extreme cases the code could even get squished into a narrow column with one word per line and we could do nothing about it?

Finally, it will also be pretty annoying to write by hand and introduce more noise in diffs - you'll need to adjust the shift whenever the variable or type name changes.